.buildkite/scripts/dra_release.sh

```diff
@@ -34,14 +33,14 @@ with_go
 with_mage
 
 if [[ "${TYPE}" == "snapshot" ]]; then
-    SNAPSHOT=true
+    export SNAPSHOT=true
 fi
 
 mkdir -p ${BASE_DIR}/reports
 ./dev-tools/dependencies-report --csv ${BASE_DIR}/reports/dependencies-${VERSION}.csv
 cd ${BASE_DIR}/reports && shasum -a 512 dependencies-${VERSION}.csv > dependencies-${VERSION}.csv.sha512
```

Comment:
I would try to use targets instead, to use everywhere the same.

Instead of 
```shell
if [[ "${TYPE}" == "snapshot" ]]; then
    export SNAPSHOT=true
fi

mkdir -p ${BASE_DIR}/reports
./dev-tools/dependencies-report --csv ${BASE_DIR}/reports/dependencies-${VERSION}.csv
cd ${BASE_DIR}/reports && shasum -a 512 dependencies-${VERSION}.csv > dependencies-${VERSION}.csv.sha512
```
I would use make as in Jenkins (https://github.com/elastic/fleet-server/blob/883cb46647979e234f70c88fd4aca1dc9f65800b/.ci/packaging.groovy#L217-L220)
```shell
if [[ "${TYPE}" == "staging" ]]; then
    make release-manager-dependencies-release
else
    make release-manager-dependencies-snapshot
fi
```

Reply:
And I think package.sh should also be updated to ensure that the same as in Jenkins is run:
https://github.com/elastic/fleet-server/blob/883cb46647979e234f70c88fd4aca1dc9f65800b/.ci/packaging.groovy#L170-L176

Instead of:
```shell
if [[ ${MATRIX_TYPE} == "staging" ]]; then
    make release
else
    make SNAPSHOT=true release
fi
```

I would use the makefile targets:
```shell
if [[ ${MATRIX_TYPE} == "staging" ]]; then
    make release-manager-release
else
    make release-manager-snapshot
fi
```